app/src/main/java/com/devindi/wallpaper/settings/model/IntField.kt

```diff
@@ -0,0 +1,19 @@
+package com.devindi.wallpaper.settings.model
+
+import android.content.SharedPreferences
+import android.support.annotation.StringRes
+
+class IntField(private val storage: SharedPreferences, @StringRes public val titleId: Int, override val key: String): SettingsField<Int> {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/f22b9ee68b14344acb1bd6cd546c2767866b4eef/app/src/main/java/com/devindi/wallpaper/settings/model/IntField.kt#L6"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>